api/v1/cluster_webhook.go

```diff
@@ -764,7 +764,7 @@ func (r *Cluster) validateBootstrapRecoverySource() field.ErrorList {
 	var result field.ErrorList
 
 	// This validation is only applicable for recovery based bootstrap
-	if r.Spec.Bootstrap == nil || r.Spec.Bootstrap.Recovery == nil || r.Spec.Bootstrap.Recovery.Source == "" {
+	if r.Spec.Bootstrap == nil || r.Spec.Bootstrap.Recovery == nil || r.Spec.Bootstrap.Recovery.Backup != nil {
```

Comment:
Why has this been changed? I don't see how it relates to the patch. Can you please comment on this @jbattiato?

Reply:
I don't know, @jlong49 did it.

Reply:
I think this change needs to be reverted.